mlflow/utils/search_utils.py

```diff
@@ -89,8 +90,14 @@ class SearchUtils:
     }
 
     @classmethod
-    def get_sql_filter_ops(cls, column, operator):
-        sql_filter_ops = {"LIKE": column.like, "ILIKE": column.ilike}
+    def get_sql_filter_ops(cls, column, operator, dialect):
+        if dialect == "mysql":
+            like_op = column.op("LIKE BINARY")
+        elif dialect == "mssql":
+            like_op = sa.collate(column, "SQL_Latin1_General_CP1_CS_AS").like
```

Comment:
`LIKE` is case-insensitive in MySQL and SQL server.

Reply:
This block enables `case_sensitive_like` for SQLite:

https://github.com/mlflow/mlflow/blob/d9489d0c61f75d763a567ade17b7168c47b4da74/mlflow/store/db/utils.py#L76-L78